Makefile

```diff
@@ -116,8 +116,8 @@ FOMANTIC_WORK_DIR := web_src/fomantic
 
 WEBPACK_SOURCES := $(shell find web_src/js web_src/css -type f)
 WEBPACK_CONFIGS := webpack.config.js
-WEBPACK_DEST := public/assets/js/index.js public/assets/css/index.css
-WEBPACK_DEST_ENTRIES := public/assets/js public/assets/css public/assets/fonts public/assets/img/webpack
+WEBPACK_DEST := public/assets/static/index.js public/assets/static/index.css
+WEBPACK_DEST_ENTRIES := public/assets/static
```

Comment:
Maybe the WEBPACK_DEST_ENTRIES can be named to "public/assets/dist" like many JS packages?

"static" is used many times in Gitea code (eg: StaticRootPath).

"dist" (or choose some new words other than "static")  might avoid some conflicts,

Reply:
Could do I guess. I see vite actually does just use `/assets/` by default, but I realy want a dedicated subdirectory.

Reply:
Maybe we could output directly into `assets` and make webpack clean everything besides `/img/`.

Reply:
Done, now it outputs into assets directly and webpack will delete everything except `img` before emit.